### PR TITLE
Add IP Fabric Integration

### PR DESCRIPTION
**Pull request by snaselj**
# Closes: #203 

## What's Changed

Merged nautobot-plugin-chatops-ipfabric to nautobot-plugin-chatops.

## TODO

- [x] Test with IP Fabric.
- [x] Add default value `True` to nautobot_config for `IPFABRIC_VERIFY`

### CODE REVIEW

File: development/nautobot_config.py [hunk 1]
```diff
@@ -148,6 +148,13 @@ def _get_bool_env(name: str):
         "slack_slash_command_prefix": os.environ.get("SLACK_SLASH_COMMAND_PREFIX", "/"),
         "webex_signing_secret": os.environ.get("WEBEX_SIGNING_SECRET"),
         "webex_token": os.environ.get("WEBEX_ACCESS_TOKEN"),
+        "ipfabric": {
+            "IPFABRIC_API_TOKEN": os.environ.get("IPFABRIC_API_TOKEN"),
+            "IPFABRIC_HOST": os.environ.get("IPFABRIC_HOST"),
+            # TBD: Add default parameter `True` once ansible will be merged
+            "IPFABRIC_VERIFY": _get_bool_env("IPFABRIC_VERIFY"),
```
Comment (snaselj) [inline development/nautobot_config.py:155]: Fix this

File: pyproject.toml [hunk 2]
```diff
@@ -17,18 +17,22 @@ include = [
 
 [tool.poetry.plugins."nautobot.workers"]
 "clear" = "nautobot_chatops.workers.clear:clear"
+"ipfabric" = "nautobot_chatops.integrations.ipfabric.worker:ipfabric"
 "nautobot" = "nautobot_chatops.workers.nautobot:nautobot"
 
 [tool.poetry.dependencies]
 # TODO Move to ~ requirements instead of ^
-python = "^3.7"
+python = "^3.7.1"
```
Comment (snaselj) [inline pyproject.toml:25]: Necessary due to ipfabric-diagrams

**Comment by pke11y** [comment]
@snaselj I tested again and the image is still not displayed in the command header.

**Comment by snaselj** [comment]
Rebased on top of develop + minor fixes

**Comment by snaselj** [comment]
> @snaselj I tested again and the image is still not displayed in the command header.

Could you pls. check it again? Static files are inside container:

```
root@8aa1a7853a48:/opt/nautobot/static/ipfabric# ls -Al
total 36
-rw-r--r-- 1 root root 19017 Jun  2 07:54 IP_Fabric_logo_small.jpg
-rw-r--r-- 1 root root 15558 Jun  2 07:54 ipfabric_logo.png
root@8aa1a7853a48:/opt/nautobot/static/ipfabric# 
```

After updating the branch to the latest version:
```
inv destroy
inv build
inv start
inv bootstrap-mattermost
```

**Comment by snaselj** [comment]
This PR is ready for review now.

**Comment by smk4664** [comment]
@pke11y Has testing been completed with this integration? I don't have an instance of IP Fabric I can test against.

**Comment by pke11y** [comment]
> @pke11y Has testing been completed with this integration? I don't have an instance of IP Fabric I can test against.

Yes @smk4664. Testing against IP Fabric was completed successfully, following the last few commits.

**Comment by snaselj** [comment]
Merger with the latest develop to resolve conflicts.

**Review by smk4664 — APPROVED** [review]
Thanks for the work and making the changes necessary! Let's get this merged!

**Pull request merged**